HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -466,6 +466,49 @@ public async Task<IActionResult> NotTakingPart_Post(IneligibleViewModel viewMode
         return RedirectToAction(forwardArgs.Action, forwardArgs.Controller, forwardArgs.Values);
     }
     
+    [HttpGet("not-participating")]
+    public IActionResult NotParticipating_Get(QuestionFlowStep? entryPoint, bool emailPreferenceSubmitted = false)
+    {
+        var questionnaire = questionnaireService.GetQuestionnaire();
+        var viewModel = new NotParticipatingViewModel()

```

Comment:
```suggestion
        var viewModel = new NotParticipatingViewModel
```
Tiny nitpick: you don't need the `()` brackets if initialising like this with `{ curly brackets }` (I do recognise you've taken this from elsewhere in the file)